src/gen9_avc_encoder.c

```diff
@@ -7331,6 +7374,10 @@ gen9_avc_vme_context_init(VADriverContextP ctx, struct intel_encoder_context *en
     struct encoder_status_buffer_internal *status_buffer;
     uint32_t base_offset = offsetof(struct i965_coded_buffer_segment, codec_private_data);
 
+    gpe = &i965->gpe_table;
+    if(!gpe)
+        goto allocate_structure_failed;
```

Comment:
It is unnecessary to check gpe against NULL pointer here because gpe is the address of i965->gpe_table.